ios/Podfile

```
@@ -34,7 +34,7 @@ target 'BluiReactNativeShowcaseDemo' do
   use_react_native!(
     :path => config[:reactNativePath],
     # Hermes is now enabled by default. Disable by setting this flag to false.
-    :hermes_enabled => flags[:hermes_enabled],
+    :hermes_enabled => flags[false],
```

Pretty sure there is no entry in the flags array called false
```suggestion
    :hermes_enabled => false,
```